.gitignore

```diff
@@ -2,3 +2,5 @@
 /node_modules
 /bin
 /dist
+yarn.lock
+package-lock.json
```

Comment:
The `package-lock.json` file must be added to the repository, because of several reasons - see the [official documentation](https://docs.npmjs.com/configuring-npm/package-lock-json.html). Valid for `yarn.lock`, too.